stl/inc/yvals_core.h

```diff
@@ -42,6 +42,7 @@
 //     (partially implemented)
 // P0898R3 Standard Library Concepts
 // P0919R3 Heterogeneous Lookup For Unordered Containers
+// P1357R1 Traits for [Un]bounded Arrays

```

Comment:
Please use #58's title here, `// P1357R1 is_bounded_array, is_unbounded_array` (we "rename" papers to make them consistent and clear).

Reply:
Done